src/app/screens/Accounts/Detail/index.tsx

```diff
@@ -238,41 +237,38 @@ function AccountDetail() {
                   />
                 </div>
               </form>
-              {lightningAddress &&
-                isAlbyOAuthAccount(authAccount?.connectorType) && (
-                  <>
-                    <MenuDivider />
-                    <div className="flex flex-col sm:flex-row justify-between items-center">
-                      <div className="sm:w-9/12 w-full">
-                        <p className="text-gray-800 dark:text-white font-medium">
-                          {t("lnaddress.title")}
-                        </p>
-                        <p className="text-gray-600 text-sm dark:text-neutral-400">
-                          {lightningAddress}
-                        </p>
-                      </div>
+              {lightningAddress && account.connectorType == "alby" && (
```

Comment:
can you add a comment above where we load the lightningAddress and here.

https://github.com/getAlby/lightning-browser-extension/pull/3180/files#diff-d7095991beda5bd1778ee83a05d8ba506d7585e20906a072039434163058722eL56

that describes that the lightningAddress is loaded from the currently active account.

and your check actually also will not work. because if I have two alby accounts in there it will show the wrong ln address.